lpa-submissions-e2e-tests/cypress/integration/invalidUrlContactUs/invalidUrlContactUs.steps.js

```diff
@@ -0,0 +1,57 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+import {
+  getContactUsLink,
+  getEmailLink,
+  getFindAboutCallChargesLink,
+} from '../../support/PageObjects/invalidUrlPage-PageObjects';
+
+const invalidUrl = '/invalid-url';
+const invalidPageTitle = 'Page not found - Appeal questionnaire - Appeal a householder planning decision - GOV.UK';
+const invalidPageHeading = 'Page not found';
+const contactUsPageUrl = '/contact-us';
+const contactUsPageTitle = 'Contact us - Appeal questionnaire - Appeal a householder planning decision - GOV.UK';
+const contactUsPageHeading = 'Contact us';
+
+Given('user has navigated to an invalid page',()=>{
+  cy.goToPage(invalidUrl, {failOnStatusCode:false});
+  cy.verifyPage(invalidUrl);
+  cy.verifyPageTitle(invalidPageTitle);
+  cy.verifyPageHeading(invalidPageHeading);
+});
+
+Given('user has navigated to contact us page',()=>{
+
+  cy.goToPage(invalidUrl, {failOnStatusCode:false});
```

Comment:
Looks like the first param should be `contactUsPageUrl` rather than `invalidUrl`?